internal/handlers/index_test.go

```diff
@@ -0,0 +1,131 @@
+package handlers
+
+import (
+	"net/http/httptest"
+	"testing"
+
+	"github.com/gofiber/fiber/v2"
+	"github.com/jiotv-go/jiotv_go/v3/internal/config"
+	"github.com/jiotv-go/jiotv_go/v3/pkg/television"
+)
+
+func TestIndexHandlerWithDefaultConfig(t *testing.T) {
+	// Save original config
+	originalCfg := config.Cfg
+
+	// Create test app
+	app := fiber.New()
+
+	tests := []struct {
+		name           string
+		defaultCats    []int
+		defaultLangs   []int
+		queryParams    string
+		expectedStatus int
+		description    string
```

Comment:
![medium](https://www.gstatic.com/codereviewagent/medium-priority.svg)

The `description` field in the `tests` struct is declared but never used within the test. It's best to remove unused code to keep the test cases clean and maintainable.